slycot/math.py

```diff
@@ -23,6 +23,107 @@
 import numpy as np
 
 
+def mb02ed(typet: str,T: np.ndarray, B: np.ndarray, n: int, k: int,  nrhs: int):
```

Comment:
```suggestion
def mb02ed(typet: str, T: np.ndarray, B: np.ndarray, n: int, k: int,  nrhs: int):
```

Reply:
May I suggest using black or some other formatter to standardize such formatting questions in the future?

Reply:
For new code yes. Changing old unrelated code in a feature PR is not so good though.
Also, in the past black did not really work well with reformatting numpy arrays. (https://github.com/python-control/python-control/issues/308)